src/main/java/module/transfer/scout/PlayerConverter.java

```diff
@@ -627,7 +615,15 @@ public final Player buildHTCopyButton(String text) {
         sc = new Scanner(row);
         sc.useDelimiter(": ");
         if (sc.hasNext()) sc.next();
-        if (sc.hasNext()) player.setExpiryDate(sc.next().trim());
+        if (sc.hasNext()) {
+            var datetime = sc.next();
+            sc.close();
+            sc = new Scanner(datetime);
+            sc.useDelimiter(" ");
+            var dateString = sc.next();
+            var timeString = sc.next();
+            player.setExpiryDate(dateString, timeString);
```

Comment:
This fails for me here:
```
java.time.format.DateTimeParseException: Text '02-05-2023 20.11' could not be parsed at index 2
	at java.base/java.time.format.DateTimeFormatter.parseResolved0(DateTimeFormatter.java:2052)
	at java.base/java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1954)
	at java.base/java.time.LocalDateTime.parse(LocalDateTime.java:494)
	at module.transfer.scout.Player.setExpiryDate(Player.java:577)
	at module.transfer.scout.PlayerConverter.buildHTCopyButton(PlayerConverter.java:625)
```
	

```
Fabrizio Consoletti [playerid=446480994]
28 years and 67 days, Next birthday: 16-06-2023

Nationality: Italia

A sympathetic guy who is temperamental and honest. Has solid experience and solid leadership. Has divine loyalty.
Owner: Olli United since 29-12-2021
TSI: 152 450
Wage: [money]154320[/money]/week
Speciality: [b]Technical[/b]
Warnings: 1
Injuries : Healthy

[table][tr][th]Form[/th][td]solid (7)[/td][/tr][tr][th]Stamina[/th][td]solid (7)[/td][/tr][/table]
[table][tr][th]Keeper[/th][td]disastrous (1)[/td][/tr][tr][th]Defending[/th][td][b]world class[/b] (13)[/td][/tr][tr][th]Playmaking[/th][td][b]magnificent[/b] (12)[/td][/tr][tr][th]Winger[/th][td][b]magnificent[/b] (12)[/td][/tr][tr][th]Passing[/th][td]inadequate (5)[/td][/tr][tr][th]Scoring[/th][td]weak (4)[/td][/tr][tr][th]Set pieces[/th][td][b]brilliant[/b] (11)[/td][/tr][/table]
Deadline: 02-05-2023 20.11
Asking price: [money]59000000[/money]
```

I think this is due to the expiry date being locale-dependent, as the date time for me appears as `02-05-2023 20.11`, rather than `dd.MM.yyyy HH:mm`

Reply:
The specialty also seems to cause troubles:

![Screenshot from 2023-05-02 19-47-19](https://user-images.githubusercontent.com/114285/235757963-0149eb9b-fe17-4ffb-862e-1a616d1a4cd2.png)

The Warning line appears to have the same line number as Specialty:

![Screenshot from 2023-05-02 19-48-15](https://user-images.githubusercontent.com/114285/235758009-bbf25e88-b9e9-4db1-998e-88ebfe39906d.png)

(I am not even sure Warnings are displayed anywhere?)

Lastly, wages are displayed with tags, not sure if that's expected?

![Screenshot from 2023-05-02 19-54-01](https://user-images.githubusercontent.com/114285/235759198-9926c2f2-2394-43ce-8daf-e1318dcdf50a.png)

Reply:
thanks a lot!!!

* Concerning Exception: 
changing the HO language didn't change the date format when I tested this. So i thought, that fixing the format could be a good idea. Now i changed this to locallized formatter - hope this will work better.
* Concerning Specialty:
this works fine for me
* Concerning Wage: this is a TODO (HO8 maybe)

Reply:
@tychobrailleur sorry i did misunderstand you (because of my poor english)
the indexRowWarning was of course wrong - repaired it now. Notes will no longer display wage (money) but the expected yellow card info (warning) instead, if player has some of them.

Wage now got an extra display component in scout panel and table

Reply:
I still don't get how the format for the deadline date is selected in Hattrick.  I have changed the language in the Preferences to English (US), French, German, Arabic, and I still get the same `DD-MM-YYYY HH.MM` format.

I have also changed Firefox settings to French, German, Arabic, and again, same format...  Without knowing what is uses, it's somewhat hard to figure out how to parse this correctly.